catboost/docs/en/concepts/algorithm-main-stages_bootstrap-options.md

```diff
@@ -20,8 +20,8 @@ Depending on the value of the `bootstrap_type` parameter, these ideas are implem
 
 * [{{ fit__bootstrap-type__Bayesian }}](#bayesian)
 * [{{ fit__bootstrap-type__Bernoulli }}](#bernoulli)
-* [{{ fit__bootstrap-type__MVS }}](#mvs) (supported only on {{ calcer_type__gpu }})
-* [{{ fit__bootstrap-type__Poisson }}](#poisson)
+* [{{ fit__bootstrap-type__MVS }}](#mvs) (supported only on CPU)
```

Comment:
MVS is actually supported by both CPU and GPU now, so this part (and `CPU-only` in the main description of `MVS`) should be actually removed.